WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -85,7 +115,50 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return 2;
+    StatsSection statsSection = [self statsSectionForTableViewSection:section];
+    id data = [self statsDataForStatsSection:statsSection];
+    
+    switch (statsSection) {
+        case StatsSectionInsightsAllTime:
+        case StatsSectionInsightsMostPopular:
+        case StatsSectionInsightsTodaysStats:
+            return 2;
+        case StatsSectionPeriodHeader:
+            return 1;
```

Comment:
Maybe, for consistency's sake, wrap up all of the `case` code segments in `{ }` ?